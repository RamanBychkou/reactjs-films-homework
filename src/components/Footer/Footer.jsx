import React from 'react';
import style from './Footer.scss';

const Footer = () => (
  <footer>
    <div className={style.container}>
      <h2>FILMS</h2>
      <p>Copyright Â© 2019 FILMS. RAMAN BYCHKOU</p>
    </div>
  </footer>
);

export default Footer;
