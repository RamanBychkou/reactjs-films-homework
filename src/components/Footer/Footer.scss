.container {
  width: 100%;
  display: flex;
  flex-flow: column wrap;
  justify-content: space-between;
  align-items: center;
  background-color: #fff;
  padding: 2rem 0;

  h2 {
    color: #0aaee4;
    font-size: 2rem;
    letter-spacing: 2px;
    font-weight: 600;
    margin: 2rem 0;
  }
}
