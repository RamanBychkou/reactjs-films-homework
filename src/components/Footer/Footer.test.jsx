import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import Footer from './Footer';

describe('Footer', () => {
  it('Footer: renders correctly', () => {
    const renderer = new ShallowRenderer();
    const result = renderer.render(<Footer />);
    expect(result).toMatchSnapshot();
  });
});
