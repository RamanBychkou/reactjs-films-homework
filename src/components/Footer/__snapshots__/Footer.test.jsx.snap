// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer: renders correctly 1`] = `
<footer>
  <div
    className="container"
  >
    <h2>
      FILMS
    </h2>
    <p>
      Copyright Â© 2019 FILMS. RAMAN BYCHKOU
    </p>
  </div>
</footer>
`;
